docs/source/other_models/pregnancy/index.rst

```diff
@@ -1,9 +1,274 @@
+
+.. list-table:: Covariates
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Covariate name
+    - Covariate ID
+    - Note
+  * - Live births (thousands)
+    - 60
+    - 
+  * - Live births by sex
+    - 1106
+    - 
+  * - Live births by sex and maternal age
+    - 2298
+    - 
+  * - Stillbirth to live birth ratio
+    - 2267
+    - Not specific to maternal age. Upper and lower bound estimates are equal to mean estimate (no uncertainty interval)
+  * - Age-specific fertility rate
+    - 13
+    - 
+  * - Total fertility rate
+    - 2363
+    - 
+
+GBD has estimated the prevalence of pregnancy (as in itermediate variable for the estimation of various outcomes) as:
+
+.. math::
+
+   (ASFR + (SBR * ASFR)) * 46/52
+
+Where :math:`ASFR` is the age-specific fertility rate, :math:`SBR` is the stillbirth to live birth ratio, and :math:`46/52` is the proportion of the year spent pregnant (40 weeks) and postpartum (6 weeks) [GBD-2019-Risk-Factors-Appendix-pregnancy]_.
+
 .. todo::
 
-   document this page
\ No newline at end of file
+   Determine the treshold of gestational age at which a loss of pregnancy is classified as a stillbirth rather than miscarriage for the GBD covariate. Standard thresholds are 20 or 24 weeks.
+
+Vivarium Modeling Strategy
+----------------------------
+
+We will model pregnancy as a characteristic of women of reproductive age in our simulations. We will inform the *incidence* of pregnancy using the age-specific fertility and stillbirth to live birth ratio covariates from GBD. We will inform the *duration* of pregnancy using the GBD 2019 exposure distribution of gestational age.
+
+.. image:: diagram.svg
+
+.. list-table:: State definitions
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - State
+    - Description
+    - Note
+  * - np
+    - Not pregnant or postpartum
+    - 
+  * - p
+    - Pregnant
+    - 
+  * - pp
+    - Postpartum
+    - 
+
+.. note::
+
+   We will use the simplifying assumption of 40 weeks gestation and six weeks postpartum for pregnancy state *intialization* in our cause model only. We could improve this assumption by using location-specific mean gestational age rather than 40 weeks as the assumed duration of pregnancy.
+
+.. list-table:: State prevalence table
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Exposure category
+    - Value
+    - Note
+  * - np
+    - :math:`1 - (ASFR + ASFR * SBR) * 46 / 52`
+    - 
+  * - p
+    - :math:`(ASFR + ASFR * SBR) * 40 / 52`
+    - 
+  * - np
+    - :math:`1 - (ASFR + ASFR * SBR) * 6 / 52`
+    -  
+
+.. list-table:: State transition data
+  :header-rows: 1
+
+  * - Source state
+    - Sink state  
+    - Transition rate
+    - Note
+  * - np
+    - p
+    - :math:`\frac{ASFR + ASFR * SBR}{prevalence_\text{np}}`
+    - 
+  * - p
+    - pp
+    - Informed by gestational age (see below section)
+    - Duration-based transition
+  * - np
+    - n
```

Comment:
Again, either a typo or I just don't understand this

```suggestion
  * - pp
    - np
```

Reply:
Yes, another typo! Thank you for your careful review